dhall-docs/src/Dhall/Docs.hs

```diff
@@ -171,39 +199,50 @@ saveHtml inputAbsDir outputAbsDir t@(absFile, _) = do
 {-| Create an index.html file on each folder available in the second argument
     that lists all the contents on that folder.
 
-    Examples:
+    For example,
 
-    >>> createIndexes $(mkAbsDir "/") [
-        , $(mkAbsFile "/a/b.txt")
-        , $(mkAbsFile "/a/c/b.txt")
-        , $(mkAbsFile "/a/c.txt")
+    @
+    createIndexes [absdir|/|]
+        [ [absfile|/a/b.t`xt|]
+        , [absfile|/a/c/b.txt|]
+        , [absfile|/a/c.txt"|]
         ]
+    @
```

Comment:
I didn't used the doctest syntax because `createIndexes` writes to file-system and I actually only want to show what is the expected behaviour of calling the function with those arguments